include/ck/tensor_operation/gpu/grid/gridwise_gemm_multiple_abd_xdl_cshuffle.hpp

```diff
@@ -657,6 +657,7 @@ struct GridwiseGemmMultipleABD_xdl_cshuffle
         auto blockwise_gemm = BlockwiseGemmXdlops_k0mk1_k0nk1_m0n0m1n1m2m3m4n2_Selector<
             BlockSize,
             ComputeDataType,
+            ComputeDataType,
```

Comment:
Was this a bug? Is this change needed here? It's not related to the PR's title.

Reply:
This is not a bug. We allow ComputeType for A/B is different.

Reply:
Well, but there seemed to be missing one compute type argument. Good you caught that :)